README.md

```diff
@@ -1 +1,146 @@
+
+## Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Personalized enumerable**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- **Ruby**
+  - If you don't have Ruby installed on your computer, follow the instructions on [this page](https://www.ruby-lang.org/en/documentation/installation/) to install it.
+
+### Setup
+
+To get a local copy up and running, follow these steps.
+
+1. Clone the repo or download the Zip folder:
+
+```sh
+git clone https://github.com/ahmedeid6842/My-Enumerable.git
+```
+
+2. Navigate to the project folder in your terminal and run the following command to install the necessary gems:
+
+```sh
+bundle install
+```
+
+3. Run the following command to start the program:
+
+```sh
+ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+1. Ensure that Ruby is installed on your system.
+2. Run the following command to start the program:
+
+```
+ruby enum.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Authors <a name="authors"></a>
+👤 **Reza Merzaie**
+
+- 👤 GitHub:   [Ali-0111](https://github.com/Gh-Reza)
+
+👤 **Ahmed Eid**
+
+- 👤 GitHub:   [AhmedEid1610](https://github.com/ahmedeid6842)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## Future Features <a name="future-features"></a>
+
+- **Add more methods**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## Contributing <a name="contributing"></a>
+
+We welcome contributions, especially bug fixes and feature additions.
+
+If you'd like to contribute to this project, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with descriptive commit messages.
+4. Push your changes to your forked repository.
+5. Submit a pull request to the main repository, explaining your changes in detail.
+
+Please adhere to the coding conventions and guidelines specified in the project.
+
+Feel free to check the [issues page](https://github.com/Gh-Reza/MorseCodeDecoding/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## Support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## Acknowledgments < name="acknowledgements"></a>
+
+I would like to thank Microverse and Ruby for making this possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] the link to the MIT license is not working as expected, please make sure it works as intended in the microverse template.